src/main/java/at/hannibal2/skyhanni/utils/ItemPriceUtils.kt

```diff
@@ -60,7 +60,11 @@ object ItemPriceUtils {
             return 7.0 // NPC price
         }
 
-        return getNpcPriceOrNull() ?: getRawCraftCostOrNull(priceSource, pastRecipes)
+        return if (priceSource == ItemPriceSource.NPC_SELL) {
+            getNpcPriceOrNull()
+        } else {
+            getNpcPriceOrNull() ?: getRawCraftCostOrNull(priceSource, pastRecipes)
+        }
```

Comment:
The duplication of `getNpcPriceOrNull()` looks a bit ugly, perhaps do this instead:
```suggestion
        return getNpcPriceOrNull()
            ?: getRawCraftCostOrNull(priceSource, pastRecipes).takeUnless { priceSource == ItemPriceSource.NPC_SELL }
```